velox/functions/prestosql/BinaryFunctions.h

```diff
@@ -436,4 +436,92 @@ struct ToIEEE754Bits32 {
   }
 };
 
+template <bool lpad, typename TOut, typename TIn>
```

Comment:
Not sure why you need TOut and TIn as template. The types are deduced from the single call made in line 517 but you could be specific with the types unless it is used elsewhere with different types.